plugins/libplugin-pay.c

```diff
@@ -234,6 +234,28 @@ static struct command_result *payment_getroute_error(struct command *cmd,
 	return command_still_pending(cmd);
 }
 
+/* Iterate through the channel_hints and exclude any channel that we are
+ * confidet will not be able to handle this payment. */
+static void payment_getroute_add_excludes(struct payment *p,
+					  struct json_stream *js)
+{
+	struct payment *root = payment_root(p);
+	struct channel_hint *hint;
```

Comment:
the `channel_hint` name is a tad bit confusing since there are also *invoice* channel hints, no? i don't have a good alternative suggestion though.

Reply:
Good point, I always call the ones in the invoice `route_hint`, and the ones that are just a collection of infos we know about channels `channel_hint`.